### ISSUE TITLE
Email configuration did not work for me

### ISSUE THREAD

**Issue by TimH54** [kmwoley/restic-windows-backup#62]
Hi!

First of all, great set of scripts to wrap around restic - I love them! 🙂

I ran into one little problem (under Windows 10). In "secrets.ps1" I had set up the info regarding my SMTP server. In particular, I had set:

`$ResticEmailConfig=@{UseSsl=$true; Port="465"}`

But email sending would not work, I would get timeouts from the mail server. Strangely enough, omitting the port made it work for me:

`$ResticEmailConfig=@{UseSsl=$true;}`

After all, 465 should be the standard port for SMTP via SSL/TLS. I hope it still uses SSL, not sure how to check that.

Cheers,
Tim

**Comment by TimH54** [comment]
Checking the headers I confirmed that it still does use SSL:

`Received: from [84.xxx.xxx.xx] (helo=xxx)
        by smtprelay06.xxx.xx with esmtpsa  (TLS1.2) tls TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384`

So, for me it's all good, but I guess worth checking so that others do not run into the same problem.

By the way, this is what I am using restic with: https://www.syncosync.org

**Comment by rawtaz** [comment]
> I would get timeouts from the mail server.

Presumably the server isn't accessible on that port then?

**Comment by TimH54** [comment]
Yeah, but 465 is the port that my provider uses: https://www.df.eu/blog/sslin-df-eu-sslout-df-eu/ I have also set that up in my email clients and it works fine there!?

So I can't really explain it, but when I leave out the port bit it works fine. (If I get the PowerShell documentation right, default port will be 25, seems that works also with SSL enabled in case of my provider.)

**Comment by rawtaz** [comment]
Would need to inspect the network traffic to determine why it fails when you have the port number configured I suppose. But if you're lucky enough that your ISP isn't blocking outgoing port 25 (very common around where I live) and it works then not much to fix :)

**Comment by kmwoley** [comment]
Is it safe to close this issue? It doesn't look like there's a code fix here needed for the scripts?
Thanks!
Kevin

**Comment by kmwoley** [comment]
Closing this issue - 1.7 introduced an updated way to send mail that hopefully resolves this issue. Please create a new issue if you find something not working.
